index.js

```diff
@@ -1,34 +1,36 @@
+'use strict';
 var Readable = require('stream').Readable;
-var _ = require('underscore');
 var request = require('request');
 
-var pagerduty = function(subdomain, token) {
+var pagerduty = function (subdomain, token, url) {
```

Comment:
@arunasank is `subdomain` no longer used?  You could get rid of it if it's no longer used.